os_stub/spdm_crypt_ext_lib/cryptlib_ext.h

```diff
@@ -488,9 +488,26 @@ extern bool libspdm_ec_check_key(const void *ec_context);
  * @retval  true   Ed public key component was set successfully.
  * @retval  false  Invalid EC public key component.
  **/
-extern bool libspdm_ecd_set_pub_key(void *ecd_context, const uint8_t *public_key,
+extern bool libspdm_ecd_set_pub_key(void **ecd_context, const uint8_t *public_key,
```

Comment:
why use double pointer `**ecd_context` ?

Reply:
Because the `EVP_PKEY_new_raw_private_key()` allocates a new EVP_PKEY.

Reference: https://www.openssl.org/docs/man1.1.1/man3/EVP_PKEY_new_raw_private_key.html

```
EVP_PKEY_new_raw_private_key() allocates a new EVP_PKEY.
EVP_PKEY_new_raw_public_key() works in the same way as EVP_PKEY_new_raw_private_key()
except that key points to the raw public key data.
```

Reply:
And I think the old ecd_context should be free in `libspdm_ecd_set_pub_key`  and `libspdm_ecd_set_pri_key` to prevent memory leak. I will add the free function in code.

Reply:
The big problem is that this API is inconsistent with other ones.

Reply:
Thanks. I have made the API consistent with others.